src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -881,7 +881,6 @@ def load_arguments(self, _):
         c.argument("grafana_resource_id", validator=validate_grafanaresourceid)
         c.argument("enable_windows_recording_rules", action="store_true")
         c.argument("enable_azure_monitor_app_monitoring", is_preview=True, action="store_true")
-        c.argument("enable_cost_analysis", is_preview=True, action="store_true")
```

Comment:
Oh, you'd better keep these parameter declarations (could remove `is_preview=True` though). The declarations in aks-preview are independent from those in official azure-cli.

Reply:
Done